scheme/common/cca/realm/realm_instanceattributes.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cca
+
+import (
+	"errors"
+
+	"github.com/veraison/corim/comid"
+)
+
+type RealmInstanceAttributes struct {
+	InstID string
+}
+
```

Comment:
Question: what is the semantics of this?

Reply:
See issue #237